s3access/s3access.py

```diff
@@ -177,21 +178,21 @@ def _check_path(path: S3Path, conditions: Dict[str, Condition]) -> bool:
     def _select_glob(self,
                      s3path: Union[str, S3Path],
                      columns: Dict[str, Type],
-                     filters: Dict[str, Condition]) -> pd.DataFrame:
+                     filters: Dict[str, Condition],
+                     reader: Reader[T]) -> T:
```

Comment:
default to Pandas?

Reply:
Internal method, will always get the reader passed in.